presto-tests/src/test/java/com/facebook/presto/tests/TestSqlFunctions.java

```diff
@@ -212,6 +200,13 @@ public void testFunctionsWithStructTypes()
         assertEquals(rows.getMaterializedRows().get(0).getFields().get(0), "United States");
     }
 
+    @Test
+    public void testCreateType()
+    {
+        assertQueryFails("CREATE TYPE testing.type.num AS integer", "Creating distinct types is not yet supported");
+        assertQuerySucceeds("CREATE TYPE testing.type.pair AS (fst integer, snd integer)");
```

Comment:
Add tests for nested type ROW type, as well we named structs that references other named struct types.